pubtools/_quay/operator_pusher.py

```diff
@@ -573,25 +592,52 @@ def build_index_images(self):
                 bundles = [self.public_bundle_ref(i) for i in g_items]
                 signing_keys = sorted(list(set([item.claims_signing_key for item in g_items])))
 
-                # build index image in IIB
                 if is_hotfix:
                     target_settings = self.target_settings.copy()
                     target_settings["iib_overwrite_from_index"] = False
                     target_settings["iib_overwrite_from_index_token"] = ""
                 else:
                     target_settings = self.target_settings
-                build_details = self.iib_add_bundles(
-                    bundles=bundles,
-                    index_image=index_image,
-                    deprecation_list=deprecation_list,
-                    build_tags=build_tags,
-                    target_settings=target_settings,
+
+                # save parameters which are used to build index image in IIB.
+                build_index_image_params.append(
+                    BuildIndexImageParam(
+                        bundles=bundles,
+                        index_image=index_image,
+                        deprecation_list=deprecation_list,
+                        build_tags=build_tags,
+                        target_settings=target_settings,
+                        tag=tag,
+                        signing_keys=signing_keys,
+                        is_hotfix=is_hotfix,
+                        hotfix_tag="" if not is_hotfix else hotfix_tag,
+                    )
                 )
-                iib_results[tag] = {
+
+        num_thread_build_index_images = self.target_settings.get("num_thread_build_index_images", 2)
```

Comment:
2 is quite low number, do you have concerns about iib performance or why you chose this number to be that low? By default number of executors for threadpool is number of cpus * 5

Reply:
I just gave a initial value, it can be overwritten in target settings. Currently, I don't know what's the appropriate value. I will ask our Operator & Foundry team and try to estimate the value later since I think it's more depending on the capacity of IIB. Anyway, I will set the initial value to 5 since we set Pub task pod cpu.limit to 1.